software/review-board.yml

```
@@ -0,0 +1,13 @@
+name: "Review Board"
+website_url: "https://www.reviewboard.org/"
+source_code_url: "https://github.com/reviewboard/reviewboard"
+description: "Extensible and friendly code review tool for projects and companies of all sizes."
+licenses:
+  - MIT
+platforms:
+  - Python
+  - Deb
+  - Docker
```

```suggestion
platforms:
  - Python
```

As far as I know no Debian package or Docker image are available. The only supported installation methods (https://www.reviewboard.org/downloads/rbtools/#other-os) is through `pip`

---

*first mistake of not reviewing properly* rip. I tried i guess.